src/lib.rs

```diff
@@ -1367,10 +1367,27 @@ impl<T: Num + Clone> Num for Complex<T> {
     type FromStrRadixErr = ParseComplexError<T::FromStrRadixErr>;
 
     /// Parses `a +/- bi`; `ai +/- b`; `a`; or `bi` where `a` and `b` are of type `T`
+    ///
+    /// `radix` must be <= 18; Larger radix would include *i* and *j* as digits,
+    /// which can not be supported.
+    ///
+    /// The conversion returns an error if 18 <= radix <= 36; it panics if radix > 36.
```

Comment:
Let's note that error/panic conditions are inherited from `T` as well.

Reply:
Ah, good point. I don't feel like I have the best formulations today, but I gave it a shot :)